examples/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -160,16 +173,16 @@ int main_(std::vector<std::string> const &args)
   // clang-format off
     desc.add_options()
         ( "help", "produce help message" )
-        ( "values", bpo::value<int>(&n_values)->default_value(50000), "number of indexable values (source) per MPI rank" )
-        ( "queries", bpo::value<int>(&n_queries)->default_value(20000), "number of queries (target) per MPI rank" )
-        ( "neighbors", bpo::value<int>(&n_neighbors)->default_value(10), "desired number of results per query" )
-        ( "overlap", bpo::value<double>(&overlap)->default_value(0.), "overlap of the point clouds. 0 means the clouds are built "
-                                                                      "next to each other. 1 means that there are built at the "
-                                                                      "same place. Negative values and values larger than two "
-                                                                      "means that the clouds are separated" )
-        ( "partition_dim", bpo::value<int>(&partition_dim)->default_value(3), "number of dimension used by the partitioning of the global "
+        ( "values", bpo::value<int>(&n_values)->default_value(50000), "Number of indexable values (source) per MPI rank." )
+        ( "queries", bpo::value<int>(&n_queries)->default_value(20000), "Number of queries (target) per MPI rank." )
+        ( "neighbors", bpo::value<int>(&n_neighbors)->default_value(10), "Desired number of results per query." )
+        ( "shift", bpo::value<double>(&shift)->default_value(1.), "Shift of the point clouds. '0' means the clouds are built "
+	                                                          "at the same place, while '1' places the clouds next to each"
+								  "other. Negative values and values larger than one "
+                                                                  "mean that the clouds are separated." )
```

Comment:
looks like string breakup is not right

Reply:
What do you mean by "right"? How do you want the breakup to be?

Reply:
But the text is justified. Just look at the file and not just the diff.